<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playMusicBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Play
Music</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopMusicBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFilesBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open
Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDirectoryBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open
Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="songsLoadedLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Songs Loaded = 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseMusicBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextMusicBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousMusicButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>88</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
